docker/frontend/Dockerfile

```diff
@@ -39,13 +39,21 @@ COPY ./install /tmp/install
 USER admin
 SHELL ["/bin/bash", "-c"]
 
-# Install the Spack workflow environment
+# Install the Spack flux environment
+RUN . /opt/spack/share/spack/setup-env.sh \
+ && cd /tmp/install \
+ && ./install_flux.sh
+
+# Install parsl into the flux environment
 RUN . /opt/spack/share/spack/setup-env.sh \
  && cd /tmp/install \
- && ./install_base.sh \
- && ./install_flux.sh \
  && ./install_parsl.sh
 
+# Re-install the Spack flux environment to rebuild view after pip install of parsl
```

Comment:
Spack itself doesn't need to be reinstalled, but the Flux spack environment does.  The reinstalled doesn't actually rebuild anything, but it rebuilds the "view" that represents the flux Spack environment so that it includes the correct versions of pip and other packages.  It's pretty confusing, and I'm not fond of doing it this way, but I haven't been able to find a more suitable solution that is reliable.